src/config.js

```diff
@@ -14,14 +14,18 @@ const rootDir = path.resolve(__dirname, '..');
 const pkg = require('../package.json');
 const { HTTPS_TYPE } = require('./utils');
 
+const zluxUtil = require('../../zlux/zlux-server-framework/lib/util.js');
```

Comment:
Please import the appropriate files inot this project.
We can name it `serverUtil`